views/jobRoles.njk

```diff
@@ -13,9 +13,26 @@
         <div class="container">
 
             <div class="mainBody">
-                <h1>Job Roles List</h1>
+                <h1 class = "m-0 p-0">Job Roles List</h1>
```

Comment:
I want the Job Roles List title in the middle of the page.

Reply:
Done